GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h

```diff
@@ -69,6 +69,9 @@ typedef void (^GIDDisconnectCompletion)(NSError *_Nullable error);
 /// The `GIDGoogleUser` object representing the current user or `nil` if there is no signed-in user.
 @property(nonatomic, readonly, nullable) GIDGoogleUser *currentUser;
 
+/// The active configuration for this instance of `GIDSignIn`.
+@property(nonatomic, nullable) GIDConfiguration *configuration;
```

Comment:
We can make this readOnly here since we don't want clients to reset it. To facilitate tests we can mark it readwrite in the GIDSignIn_Private.h.

Reply:
I've made this property readwrite to preserve the ability for developers to specify and change config at runtime.  Do you see a problem with this?